### DISCUSSION TITLE
I migrated my application using Quartz from net full framework to net core 8. I am using Quartz v3.11.0 in my application. Is that meant to work?

### DISCUSSION THREAD

**Discussion by iuliusandu75** [quartznet/quartznet#2489] [General] [answered]
I ran tests and everything runs perfectly. I wonder if that is supposed to work though since I believe Quartz 3.11 is only meant to work up to .net 6? I know Quartz 4.x is coming and that will target net core 8 but it's not out yet so I cannot use it yet.

**Comment by lahma** [comment, marked as answer]
Libraries targeting net6.0 will also work on later runtimes like net7.0, net8.0 and net9.0.

**Reply by iuliusandu75** [reply]
Thanks Marko. That's good to know.
